scripts/build/load-packages-into-dark-canvas

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+. ./scripts/devcontainer/_assert-in-container "$0" "$@"
```

Comment:
As a follow-up, would be great to get this shared code split into a `./scripts/run-local-exec` script.